lib/mock-nexmo.js

```diff
@@ -54,7 +54,8 @@ function MockNexmo(log, config) {
 
         // HACK: Enable remote tests to see what was sent
         mailer.sendMail({
-          from: `sms.${senderId}@restmail.net`,
+          from: config.smtp.sender,
+          sender: config.smtp.sender,
```

Comment:
> This is how [it's done over here](https://github.com/mozilla/fxa-auth-server/blob/3f7ed68e0958b143625570e927ebdba4c15d0c8d/lib/senders/email.js#L228:L229), so I kept both fields.

If it works with just `from` let's stick to just that I reckon. In the hierarchy of good reasons for doing stuff, "because it's done like that elsewhere" seems pretty flimsy. That's how cargo cults begin! :smile: